### PR TITLE
Viewing monitor results

### PR DESCRIPTION
**Pull request by claireriley**
This PR includes the revamped "Viewing monitor results" page (basically entirely rewritten). I moved the "Monitor dashboard" section to that page as I felt it made more sense to lead with that. I also updated for branded terms (see confluence). Lastly, noticed I had previously used "re-run" instead of "rerun" in "Setting up a monitor", so I fixed that.

Oh and my first gif! (let me know if it looks good, I couldn't get the 1px border to work @SueSmith)

@ArjunSingh-PM let me know if I'm missing anything or what pieces could be better worded, was working off of the notes from syncs we had a bit ago.

### CODE REVIEW

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 1]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
-    name: "Next Steps"
-  - type: link
-    name: "Intro to collection runs"
-    url: "/docs/postman/collection-runs/intro-to-collection-runs/"
 
 warning: false
 ---
 
-You can set up monitors to send daily or weekly emails with a summary for all your monitors. You can [turn off email notifications](/docs/postman/monitors/setting-up-monitor/) in the settings.
+Your Postman Dashboard allows you to track the health and performance of your APIs. With the Dashboard, you can stay up to date on what's happening across multiple monitors, as well as dive into an individual monitor's performance over time.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:22]: think the sentence might read a wee bit better with "and" instead of "as well as"

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 2]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+
+## Viewing individual monitors
+
+You can view each monitor in more detail by selecting it from the Dashboard.
+
+### Monitor summary
+
+You can use the **Monitor Summary** to see how your APIs have performed over time. Each monitor run is represented by a bar in the graph.
+
+The upper section charts your monitor's average response time for each run, while the lower section visualizes the percentage of failure for each run. To view the exact response time and failed percent, you can hover over each run individually.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:70]: "percentage failures" maybe in the first sentence here?

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 3]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+
+[![monitor summary](https://assets.postman.com/postman-docs/individual-monitor.jpg)](https://assets.postman.com/postman-docs/individual-monitor.jpg)
+
+### Request split
+
+You can use **Request Split** to see how the response time varies for all requests made in a given run. To break this down into individual requests, you'll want to utilize [Filters](#Filters).
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:76]: I think I'd reword "you'll want to" - even just "you can" or whatever

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 4]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+
+[![monitor summary](https://assets.postman.com/postman-docs/request-split.jpg)](https://assets.postman.com/postman-docs/request-split.jpg)
+
+### Filters
+
+You can use filters to identify recurring patterns in your monitoring runs, by comparing requests, run types, results, and regions
+
+[![monitor filters](https://assets.postman.com/postman-docs/filter-example.gif)](https://assets.postman.com/postman-docs/filter-example.gif)
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:84]: gif looks great and aye let's not bother about borders on these

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 5]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+> You can **Clear Filters** to return to your original dashboard view.
+
+#### Filtering by request
+
+You can filter by request to compare an individual request's response time in different runs. To do so, click to open the drop-down menu **All Requests** next to **Filter by**, then select your request.
+
+#### Filtering by type
 
-In addition, you can receive important notifications (both [in-app](/docs/postman/notifications) and email) if a monitor fails.
+You can filter by run type to compare how the response time changes between manual runs, scheduled runs, and webhook runs. To do so, click to open the drop-down menu **Type: All**, then select the type of run you'd like to analyze further.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:94]: same again re "to do so" feels like it might be unfamiliar to people speaking english as a second language

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 6]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+
+You can use filters to identify recurring patterns in your monitoring runs, by comparing requests, run types, results, and regions
+
+[![monitor filters](https://assets.postman.com/postman-docs/filter-example.gif)](https://assets.postman.com/postman-docs/filter-example.gif)
+
+> You can **Clear Filters** to return to your original dashboard view.
+
+#### Filtering by request
+
+You can filter by request to compare an individual request's response time in different runs. To do so, click to open the drop-down menu **All Requests** next to **Filter by**, then select your request.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:90]: maybe lose "to do so"

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 7]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+> Manual runs are initiated in the Postman Dashboard. Scheduled runs are initiated by the schedule you set when creating or editing your monitor. Webhook runs are initiated by integrations you've created.
 
-## Monitors page
+#### Filtering by run result
 
-The [Monitors](https://monitor.getpostman.com/) page lists all your monitors (both team and private).
+Each run is labeled based on its result:
 
-[![monitors page](https://assets.postman.com/postman-docs/WS-monitoring-results-page-1.png)](https://assets.postman.com/postman-docs/WS-monitoring-results-page-1.png)
+* Successful: Your monitor completed the run with no issues and passed all tests.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:102]: maybe bold "successful" etc in each of these?

Comment (SueSmith) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:102]: some potentially awkward stuff to word expressed nice and clearly here 👍

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 8]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+[![test results](https://assets.postman.com/postman-docs/test-results-2.jpg)](https://assets.postman.com/postman-docs/test-results-2.jpg)
 
-These visuals are a great way to measure performance improvements when you’ve made changes to your infrastructure.
+### Console log
 
-[![monitor perf](https://assets.postman.com/postman-docs/WS-monitor-perf-page-1.png)](https://assets.postman.com/postman-docs/WS-monitor-perf-page-1.png)
+You can view the **Console Log** below the monitor summary.
 
-The results section shows request-level details: test results, response code, response time, and the response size. Additionally, you can filter by region if you set up [monitors in multiple regions](/docs/postman/monitors/intro-monitors/#monitoring-resources-in-multiple-regions).
+This console logs in detail run events and 'console.log' statements that ran as part of your pre-request and test scripts. Run events include when Postman is preparing your run, running, rerunning ([if applicable](/docs/postman/monitors/setting-up-monitor/#using-retry-on-failure)), and the run result.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:123]: maybe "logs detail for run events" or something?

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 9]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+[![console log](https://assets.postman.com/postman-docs/console-log-2.jpg)](https://assets.postman.com/postman-docs/console-log-2.jpg)
 
-You can also view the log when you click the **Console Log** tab.
+ You can use this console to both troubleshoot issues and learn more about an individual run's behavior.
 
-[![view logs](https://assets.postman.com/postman-docs/59042622.png)](https://assets.postman.com/postman-docs/59042622.png)
+## Next steps
 
-The console log prints a detailed log of run events and 'console.log' statements that ran as part of your pre-request and test scripts. You can use them to [diagnose failures](/docs/postman/monitors/troubleshooting-monitors).
+Learn how to [troubleshoot your monitors](/docs/postman/monitors/troubleshooting-monitors/) and check out [Postman monitoring's FAQs](/docs/postman/monitors/faqs-monitors/).
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:131]: think I'd lose the 's in the last link here

File: src/pages/docs/postman/monitors/viewing-monitor-results.md (hunk 8 above)
Comment (ArjunSingh-PM) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:123]: Probably - "This section logs monitor run details along with the 'console.log' (https://learning.postman.com/docs/postman/sending-api-requests/debugging-and-logs/) statements that run as part of your pre-request and test scripts. Run details specify the various stages of a monitor run such as preparing run, running, rerunning ([if applicable](/docs/postman/monitors/setting-up-monitor/#using-retry-on-failure)), and the run result along with error and test failure details.

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 10]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
-Red indicates failing tests. Blue indicates the total response time of all the requests over time.
+[![test results](https://assets.postman.com/postman-docs/test-results-2.jpg)](https://assets.postman.com/postman-docs/test-results-2.jpg)
 
-These visuals are a great way to measure performance improvements when you’ve made changes to your infrastructure.
+### Console log
 
-[![monitor perf](https://assets.postman.com/postman-docs/WS-monitor-perf-page-1.png)](https://assets.postman.com/postman-docs/WS-monitor-perf-page-1.png)
+You can view the **Console Log** below the monitor summary.
 
-The results section shows request-level details: test results, response code, response time, and the response size. Additionally, you can filter by region if you set up [monitors in multiple regions](/docs/postman/monitors/intro-monitors/#monitoring-resources-in-multiple-regions).
```
Comment (ArjunSingh-PM) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:67]: We should add a statement to specify the following (please feel free to reword this):

If your monitor is configured to run in multiple regions, you can view the test results or console logs for a particular region by selecting that region from the dropdown at the right of the 'console log' tab

File: src/pages/docs/postman/monitors/viewing-monitor-results.md (hunk 2 above)
Comment (claireriley) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:70]: Either way sounds awkward to me. Thoughts on the following?

> The upper section charts your monitor's average response time for each run, while the lower section visualizes the percentage of failed tests for each run across all regions.

File: src/pages/docs/postman/monitors/viewing-monitor-results.md (hunk 5 above)
Comment (claireriley) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:94]: had no idea I used that phrase so often in this. all removed!

File: src/pages/docs/postman/monitors/viewing-monitor-results.md (hunk 8 above)
Comment (claireriley) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:123]: Used that and changed the last word to "information" as using "details" twice sounds awkward to me, not sure how else to get around it.

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 11]
```diff
@@ -166,16 +166,6 @@ There are a few differences between running collections in a Postman monitor and
 
 * Monitors require all URLs to be publicly available on the internet as they run in the Postman cloud. A monitor cannot directly access your localhost or run requests behind a firewall. However, to overcome this issue, static IPs are available on [Postman Business and Enterprise plans](https://www.postman.com/pricing).
 
-## Monitor dashboard
```
[the remark this answers is not in the document]
Comment (claireriley) [reply src/pages/docs/postman/monitors/intro-monitors.md:169]: Ah thanks for catching that!

File: src/pages/docs/postman/monitors/viewing-monitor-results.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (claireriley) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:22]: Okay kind of combined the two let me know what you think

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 12]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
+
+* [Next steps](#next-steps)
+
+## Viewing monitors in the Dashboard
+
+Each workspace has its own monitoring space within the Postman Dashboard, which you can navigate to by opening your [Dashboard](https://go.postman.co/) and selecting your workspace > **Monitors**. Monitors in team workspaces are visible to all members of the workspace.
+
+The Dashboard provides a high-level overview of the monitors you have available in your workspace, including status, average success rate, and average response time within the given timeframe.
+
+[![monitoring dashboard](https://assets.postman.com/postman-docs/monitor-dashboard1.jpg)](https://assets.postman.com/postman-docs/monitor-dashboard.jpg)
```
[the remark this answers is not in the document]
Comment (claireriley) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:56]: Odd both links show the same image of the dashboard for me—I've changed it to only be one as the duplication was accidental. There was previously a section that showed the summary email, but I removed that here as it didn't flow into the workspace-level monitor view

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 13]
```diff
@@ -20,57 +15,117 @@ contextual_links:
...
-## Monitors page
+#### Filtering by run result
 
-The [Monitors](https://monitor.getpostman.com/) page lists all your monitors (both team and private).
+Each run is labeled based on its result:
 
-[![monitors page](https://assets.postman.com/postman-docs/WS-monitoring-results-page-1.png)](https://assets.postman.com/postman-docs/WS-monitoring-results-page-1.png)
+* Successful: Your monitor completed the run with no issues and passed all tests.
+* Failure: Your monitor completed the run, however one or more tests failed.
+* Error: Your monitor was unable to complete its run due to an error. Errors are caused by Postman's network and are not a result of the monitor you've built. If you experience this, see [Postman's status page](https://status.postman.com/) or contact [Postman support](https://support.getpostman.com/).
```
[the remark this answers is not in the document]
Comment (claireriley) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:104]: Again, only working off of notes from our sync here and I believe network error was the only one noted then. Fixed

File: src/pages/docs/postman/monitors/viewing-monitor-results.md (hunk 10 above)
Comment (claireriley) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:67]: added this as a note in both sections

File: src/pages/docs/postman/monitors/viewing-monitor-results.md [hunk 14]
```diff
@@ -20,57 +15,134 @@ contextual_links:
...
+You can filter by run result to compare how your runs with the same result have differed. Click to open the drop-down menu **Run result: All**, then select one or more types of run results to view.
 
-In addition, you can receive important notifications (both [in-app](/docs/postman/notifications) and email) if a monitor fails.
+#### Filtering by region
 
-[![email summary of monitors](https://assets.postman.com/postman-docs/monitoring-email-results1.png)](https://assets.postman.com/postman-docs/monitoring-email-results1.png)
+You can filter by [region](/docs/postman/monitors/setting-up-monitor/#adding-regions) to compare how runs within different regions have varied. Click to open the drop-down menu **All Regions**, then select a region to view.
 
-## Monitors page
+#### Filtering by formula
```
Comment (claireriley) [inline src/pages/docs/postman/monitors/viewing-monitor-results.md:115]: @ArjunSingh-PM added this section I forgot to include earlier

Comment (ArjunSingh-PM) [reply src/pages/docs/postman/monitors/viewing-monitor-results.md:115]: We can add that this filter is only visible/applicable if the monitor is run across multiple regions. Also, this filter is only visible if "All Regions" is selected.

**Comment by claireriley** [comment]
@SueSmith can you include this in your next push to prod? 😄
